CHANGELOG_PENDING.md

```diff
@@ -1,3 +1,7 @@
 # Unreleased Changes
 
 ## vX.X
+
+#### General
+
+- [#1810](https://github.com/livepeer/go-livepeer/pull/1810) Display "n/a" in CLI when max gas price isn't specified (@kyriediculous)
```

Comment:
Let's update this based on format description in the [contributing](https://github.com/livepeer/go-livepeer/blob/master/doc/contributing.md#changelog) guide.